src/Raven.Client/Documents/Operations/Attachments/AttachmentDetails.cs

```diff
@@ -1,5 +1,18 @@
-﻿namespace Raven.Client.Documents.Operations.Attachments
+﻿using System.Collections.Generic;
+
+namespace Raven.Client.Documents.Operations.Attachments
 {
+    public class AttachmentsDetails
+    {
+        public List<AttachmentAdvancedDetails> AttachmentsMetadata;
+    }
+
+    public class AttachmentAdvancedDetails : AttachmentDetails
```

Comment:
What is this for?
And Why is AttachmentDetails recursive?

Reply:
done